fiftyone/zoo/models/manifest-torch.json

```diff
@@ -1742,31 +1742,28 @@
         },
         {
             "base_name": "complex-yolo-v3-torch",
-            "base_filename": "complex_yolov3",
+            "base_filename": "yolov3_ckpt_epoch-298.pth",
             "version": null,
             "description": "Complex extension of YOLO object detection model <https://arxiv.org/abs/1803.06199>`_ ",
             "source": "https://github.com/ghimiredhikura/Complex-YOLOv3",
             "size_bytes": 353976522,
             "manager": {
                 "type": "fiftyone.core.models.ModelManager",
                 "config": {
-                    "url": "https://github.com/ghimiredhikura/Complex-YOLOv3/blob/master/config/complex_yolov3.cfg"
+                    "google_drive_id": "1e7PCqeV3tS68KtBIUbX34Uhy81XnsYZV"
                 }
             },
             "default_deployment_config_dict": {
                 "type": "fiftyone.utils.complex_yolo.TorchComplexYOLOv3Model",
                 "config": {
-                    "weights_base_gid": "1e7PCqeV3tS68KtBIUbX34Uhy81XnsYZV",
-                    "weights_base_filename": "yolov3_ckpt_epoch-298.pth",
-                    "config_base_url": "https://raw.githubusercontent.com/ghimiredhikura/Complex-YOLOv3/master/config/complex_yolov3.cfg",
+                    "config_url": "https://raw.githubusercontent.com/ghimiredhikura/Complex-YOLOv3/master/config/complex_yolov3.cfg",
                     "config_base_filename": "complex_yolov3.cfg",
-                    "classes": "Car, Pedestrian, Cyclist",
                     "entrypoint_fcn": "",
-                    "output_processor_cls": "fiftyone.utils.torch.DetectorOutputProcessor",
-                    "labels_path": "{{eta-resources}}/ms-coco-labels.txt",
-                    "image_size": null,
-                    "image_min_dim": 224,
-                    "embeddings_layer": null
+                    "output_processor_cls": "fiftyone.utils.torch.NullOutputProcessor",
+                    "labels_string": "Car,Pedestrian,Cyclist",
```

Comment:
Better to reuse the existing parameters declared by `TorchImageModelConfig` that we're subclassing from here.

Reply:
I added `NullOutputProcessor` to make it a bit more explicit that our subclass doesn't actually use the `OutputProcessor` interface as it is currently implemented.

We could put the output processing logic *into* a new `OutputProcessor` subclass, but this model doesn't use the builtin `predict_all()` implementation, so there's not much value in following the `OutputProcessor` interface currently.